src/components/Preferences/LocalIdentities.js

```diff
@@ -222,16 +256,48 @@ LocalIdentities.propTypes = {
   onModifyEvent: PropTypes.func,
   onToggleAddress: PropTypes.func.isRequired,
   onToggleAll: PropTypes.func.isRequired,
+  opened: PropTypes.bool.isRequired,
+  setOpened: PropTypes.bool.isRequired,
 }
 
 LocalIdentities.defaultProps = {
   onModifyEvent: () => null,
 }
 
+const ModalTitle = styled.h1`
+  font-size: 22px;
+`
+
+const ModalText = styled.p`
+  margin: 20px 0 20px 0;
+`
+
+const ModalControls = styled.div`
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: 1fr 1fr;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: flex;
+      justify-content: flex-end;
+    `
+  )}
+`
+
 const StyledCheckbox = styled(Checkbox)`
   margin-right: ${3 * GU}px;
 `
 
+const RemoveButton = styled(Button)`
+  ${breakpoint(
+    'medium',
+    `
+      margin-left: 20px;
```

Comment:
We are using a system of _Grid Units_ (@bpierre authored it) to make it easier and consistent for us to manage spacing. It has `8px` as base: `margin-left: ${2.5 * GU}px;`